### ISSUE TITLE
Reduce the number of published files from alevin-fry output

### ISSUE THREAD

**Issue by tomsing1** [nf-core/scrnaseq#178]
### Description of feature

I am using the alevin-fry quantitation method. For downstream analysis, I am mainly interested in the final count matrix, e.g. 

1. The content of the `af_quant/alevin directory` and
2. The `af_quant/quant.json` file

Right now, the workflow publishes lots of other files - some of them very large - as well, e.g. the `af_map` output directory or the `af_quant/alevin/map.collated.rad` which can be tens of gigabytes in size for large experiments.

It would be great to be able to whittle down the published files to reduce the size of the pipeline's output. (After all, the intermediate files are still available in the working directory.) 

For example, I am running nextflow on AWS Batch with an S3 bucket as the publish directory. It takes many times longer to copy the output files to the bucket than to run the actual workflow (because the publishing is not parallelized*.)

*If there is a way to speed this up, I would love to learn!

**Comment by grst** [comment]
I agree we don't need all these intermediate files. Happy to accept a PR! 

We could add an option `--save_align_intermediates` as in the rnaseq workflow: 
![image](https://user-images.githubusercontent.com/7051479/202483911-5eac4998-f1e2-419f-a5ff-b2e5b3a9d80a.png)

**Comment by grst** [comment]
Depends on #361

**Comment by grst** [comment]
Alevin-fry has been replaced with simpleaf quite some time ago. 
Let's check if there are still some large intermediate files published by default.

**Comment by grst** [comment]
Checked, and this is still the case, but I think the `.rad` file is the only big one, but it appears in two locations: 

<img width="2321" height="936" alt="Image" src="https://github.com/user-attachments/assets/ef22f148-afb4-498f-a8f4-cc8902da7d57" />

<img width="2232" height="357" alt="Image" src="https://github.com/user-attachments/assets/716cbc39-9875-42af-ac07-5f757367ebc5" />

Additionally, the cellbender checkpoint could also be optional (but this is independent of alevin-fry):

<img width="2303" height="305" alt="Image" src="https://github.com/user-attachments/assets/bb5cd9cc-f0ce-49d8-9ac1-be6b45d83c48" />
